specs/specs.bs

```diff
@@ -1034,14 +1033,14 @@ Also:
 
 ## Geometry templates 
 
-CityGML's "ImplicitGeometries", better known in computer graphics as *templates*, are one method to compress files since the geometries (such as benches, lamp posts, and trees), need only be defined once. 
+CityGML's "ImplicitGeometries", better known in computer graphics as *templates*, are one method of compressing files since the geometries (such as benches, lamp posts, and trees) need be defined only once. 
```

Comment:
```suggestion
CityGML's "ImplicitGeometries", better known in computer graphics as *templates*, are one method of compressing files since the geometries (such as benches, lamp posts, and trees) need to be defined only once. 
```